### ISSUE TITLE
Mechjeb begins gravity turn at 500 meters regardless of setting.

### ISSUE THREAD

**Issue by XOIIO** [MuMech/MechJeb2#616]
Mechjeb's ascent guidance is essentially useless as of right now as I have tested on various ships and with all manner of ascent profiles, mechjeb instantly goes into gravity turn at about 500 meters or so, and is unable to leave the atmosphere save for one extremely lightweight rocket I made. All other functions that I have been able to test (rendezvous, executing next node, matching speed, plotting intercept courses) are working fine. Mechjeb is my only mod and a fresh install did not work (on version 1.0.2, this was installed on a fresh copy of KSP the first time)

Has this been addressed in the dev version by any chance?

Log file here: https://www.dropbox.com/s/9hpvcygfdhi7bt9/MechJeb2.log?dl=0

**Comment by BloodyRain2k** [comment]
Even IF it should start the gravity turn that early (which is normal btw, I've set mine to start even earlier and I can launch very big rockets no probs at all) should it still be able to reach orbit.

So I'd say you have some borked settings, does it turn too much towards the horizon when starting to turn or does your rocket simply pancake (lose control)?
Because if that's the case you most likely just have a bad design.

The new aero is like playing with FAR, use fins and control surfaces or you'll have a very hard time.
Also try a dev build if you didn't already http://jenkins.mumech.com/job/MechJeb2/

**Comment by XOIIO** [comment]
It turns too flat too early, no matter how high I set the start of the turn or arrange the ascent profile, it's usually horizontal within the atmosphere (third layer)

I've also used mechjeb to launch massive rockets, I'll try that dev build.

**Comment by XOIIO** [comment]
I figured it out, in the dev build there is a velocity slider, but in the main release there is not, once I whacked the velocity slider up to max with the same ascent profile, everything is working fine and dandy, I am much farther from vertical in the last layer of the atmosphere. The default ascent profile isn't doing the trick but it's definitely working better now.

I'm guessing the slider got left out but the code to base the ascent gravity turn on speed remained.
